databricks/koalas/namespace.py

```diff
@@ -1446,7 +1449,7 @@ def read_sql(sql, con, index_col=None, columns=None, **options):
 
 def to_datetime(
     arg, errors="raise", format=None, unit=None, infer_datetime_format=False, origin="unix"
-):
+) -> Union[DataFrame, Series, pd.Timestamp, pd.DatetimeIndex]:
```

Comment:
Could you remove type annotations for functions which are intended to work with both Koalas and pandas for now?

Currently it fails due to the type inference:

```py
>>> kdf = ks.DataFrame({"year": [2015, 2016], "month": [2, 3], "day": [4, 5], 'ymd': ['20150204', '20160305']})
>>> kdf
   year  month  day       ymd
0  2015      2    4  20150204
1  2016      3    5  20160305
>>> kdf.ymd.transform(ks.to_datetime)
Traceback (most recent call last):
...
TypeError: issubclass() arg 1 must be a class
```

cc @HyukjinKwon

Reply:
Good catch! Removed.